core/common/src/MapLayerSettings.ts

```diff
@@ -380,3 +347,140 @@ export class MapLayerSettings {
     return children;
   }
 }
+
+/** JSON representation of a [[BaseMapLayerSettings]].
+ * @beta
+ */
+export interface BaseMapLayerProps extends MapLayerProps {
+  provider?: BackgroundMapProviderProps;
+}
+
+/** A [[MapLayerSettings]] that can serve as the base layer for a [[MapImagerySettings]].
+ * The base layer supports all of the same options as any other layer, but also allows for simplified configuration based
+ * on a small set of known supported [[BackgroundMapProviders]] like [Bing Maps](https://www.microsoft.com/en-us/maps).
+ * If the base layer was configured from such a provider, that information will be preserved and can be queried; this allows
+ * the imagery provider and/or type to be easily modified.
+ * @see [[MapImagerySettings.backgroundBase]].
+ * @beta
+ */
+export class BaseMapLayerSettings extends MapLayerSettings {
+  private _provider?: BackgroundMapProvider;
+
+  /** The provider from which this base layer was configured, if any. */
+  public get provider(): BackgroundMapProvider | undefined { return this._provider; }
+
+  /** Create a base layer from its JSON representation.
+   * TODO: This, MapLayerSettings.fromJSON, and MapSubLayerSettings.fromJSON should never return undefined.

```

Comment:
@mdastous-bentley probably a separate PR?

Reply:
So make MapLayerSettings.fromJSON/ BaseMapLayerSettings.fromJSON defaults to bing right?

Im unsure what aould be proper defaults for MapSubLayerSettings.fromJSON, we might end up with subLayers with empty names.  I guess MapImagerySettings.ctor can check if name if empty, dont add itto _backgroundLayers/_overlayLayers.  What do you think?

Yes that can made part of another PR.

Reply:
No, I'm not suggesting we produce a default if we can't make a layer out of the inputs. I'm suggesting that, for example, MapLayerSettings.fromJSON takes an argument that meets the minimum requirements `{ url: string, name: string }` (neither can be undefined) such that you never bail like you currently do:
```
  public static fromJSON(json?: MapLayerProps): MapLayerSettings | undefined {
    if (json === undefined || json.url === undefined || undefined === json.name)
      return undefined;
```